index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>myPage</title>
```

Comment:
`favicon` 을 추가해보는 건 어떤가요!? 
<img width="71" alt="스크린샷 2023-06-05 오후 9 53 47" title="쨔쟌!" src="https://github.com/JAVA-ggwak-java/kyeongseon-repo/assets/47032054/0fdf47a6-d386-4966-ba90-ebf7aa6b8663">

```suggestion
  <title>myPage</title>
  <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
```

![baby_apeach](https://github.com/JAVA-ggwak-java/kyeongseon-repo/assets/47032054/0c524cd7-7d3d-49ca-ad87-8ca653d191d6)

Reply:
헐...ㅜㅜ 너무 좋은 생각이에요! 감사해요 바로 실행하겠습니다~